command/job_dispatch.go

```diff
@@ -161,6 +176,12 @@ func (c *JobDispatchCommand) Run(args []string) int {
 	// eval.
 	evalCreated := resp.EvalID != ""
 
+	// See if dispatched job was skipped due to idempotency.
+	if !evalCreated && idempotencyToken != "" {
+		c.Ui.Output(fmt.Sprintf("Job %q already dispatched with idempotency token %q.", resp.DispatchedJobID, idempotencyToken))
+		return 0
+	}
+
```

Comment:
Considering that this is intended for failure retries primarily, it may make sense to just print the existing job info which is what I assume the user wants, but without displaying the eval info. What do you think?

We can indicate that the job was already started if that's helpful for users in addition to displaying job info. AFAIK, other APIs (e.g. EC2 and Google Cloud APIs) returned results don't add such indicator - but I could be wrong.